subprojects/docs/src/docs/release/notes.md

```diff
@@ -339,6 +339,25 @@ one of the addresses of the current machine's network interfaces.
 Similarly, the new Gradle property `org.gradle.debug.host` now enables [running the Gradle process with the debugger server](userguide/troubleshooting.html#sec:troubleshooting_build_logic)
 accepting connections via network on Java 9+.
 
+#### Introduced `projectComplete()` dependency for creating unit-like test suites
+
+The [JVM test suite](userguide/jvm_test_suite_plugin.html) `dependencies` block now has support for depending on the complete view of the current project during compile-time. Previously it was only possible to depend on the current project's API. This allows declaration of unit-like test suites which have access to dependencies of project internals which are not declared on the `api` or `compileOnlyApi` configurations. 
```

Comment:
> which have access to dependencies of project internals which are not declared on the `api` or `compileOnlyApi` configurations. 

This is hard to parse for me due to the double use of `which`. I can't think of a good improvement though, perhaps @nathan-contino has some ideas.